app/views/shared/_part.html.erb

```diff
@@ -17,6 +17,13 @@
           <span class="form-wrapper">
             <%= f.text_field :title, disabled: @resource.locked_for_edits?, class: "title form-control" %>
           </span>
+          <% if f.object.errors[:title].any? %>
+            <ul class="help-block error-block">
+              <% f.object.errors[:title].each do |error| %>
+                <li><%= error %></li>
+              <% end %>
+            </ul>
+          <% end %>
```

Comment:
Should this be using the new helper?

Reply:
Also will the JS be able to clear these?

Reply:
Rebased and updated to use the `FormHelper`